common/config/rush/command-line.json

```diff
@@ -31,6 +31,25 @@
       "shellCommand": "node common/scripts/ensure-consistent-versions.js",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "bulk",
+      "name": "build:public",
+      "summary": "Preprocess and build the non-beta version.",
```

Comment:
nitty nit: Reword to be positive (what's non-beta, does it mean nightly?): "Build the @public variant of all packages"

Let's also consistently choose a word to refer to this: `version` vs `variant` vs `flavour`.
I'd argue against `version` (version refers to SemVer). I prefer `variant` over `flavour` because I don't want to get into endless fights about British vs American spelling :P

Reply:
Along those lines, could we also rename this command to `build-variant:public` and `test-variant:public`? By itself, `build:public` is a bit confusing (it makes me think there's a private thing I could build instead).

Reply:
For build:public one, maybe we should call it build:prod? I do think build:public sounds like there is a private version, and our beta is still "public", so maybe prod or ga would be something more accurate?

Reply:
I come up with another term build:stable that sounds like a good term for GA, so we might have alpha, beta then stable for GA